### ISSUE TITLE
Upload ZIM files as soon as they have been created

### ISSUE THREAD

**Issue by kelson42** [openzim/zimfarm#287]
In the current implementation, ZIM files are uploaded - all together - at the end of the scraping process. This means that if the process creates 10 ZIM files and takes 1 month, nothing will be online before one month. It would bring value to the users if the ZIM files would be uploaded just immediately after their creation. This would also probably speed-up the overall process as most uploads will happen in parallel of the scraper process

**Comment by rgaudin** [comment]
Duplicate of #286
